src/main/java/com/bio4j/angulillos/Property.java

```diff
@@ -6,17 +6,13 @@
   A property of the [Element](TypedElement.java.md) `N`, with value type `V`.
 */
 interface Property <
-  // the element type
-  N extends TypedElement<N,NT,G,I,RV,RE>, NT extends TypedElement.Type<N,NT,G,I,RV,RE>,
-  // the property type and its value type
-  P extends Property<N,NT,P,V,G,I,RV,RE>, V,
-  // graph
-  G extends TypedGraph<G,I,RV,RE>, I extends UntypedGraph<RV,RE>, RV,RE
+  ET extends TypedElement.Type,
```

Comment:
What's the point in adding parameters to `TypedElement.Type` if it doesn't need any?